apps/hash-ai-worker-ts/package.json

```diff
@@ -35,9 +35,9 @@
     "sanitize-html": "NODE_ENV=development yarn exe ./scripts/sanitize-html.ts",
     "sentry:sourcemaps": "sentry-cli sourcemaps inject --org hashintel --project hash-temporal-worker-ai ./dist && sentry-cli sourcemaps upload --org hashintel --project hash-temporal-worker-ai ./dist",
     "start": "NODE_ENV=production NODE_OPTIONS=--max-old-space-size=2048 node ./dist/main.js",
-    "start:healthcheck": "wait-on --timeout 120000 http-get://localhost:4100/health",
+    "start:healthcheck": "wait-on --timeout 120000 http-get://0.0.0.0:4100/health",
     "start:test": "NODE_ENV=test NODE_OPTIONS=--max-old-space-size=2048 node ./dist/main.js",
-    "start:test:healthcheck": "wait-on --timeout 120000 http-get://localhost:4100/health",
+    "start:test:healthcheck": "wait-on --timeout 120000 http-get://0.0.0.0:4100/health",
```

Comment:
I don't think this should be `0.0.0.0` but either `localhost` or `127.0.0.1`

Reply:
`localhost` does not work. I can change it to `127.0.0.1`, `0.0.0.0` is just what we use in other healthcheck scripts

Reply:
[Previous Slack discussion](https://hashintel.slack.com/archives/C02K2ARC1BK/p1695142080387669?thread_ts=1695136207.175219&cid=C02K2ARC1BK) – it looks like `127.0.0.1` may not work either. We'll see shortly...

Reply:
It did not work – https://github.com/hashintel/hash/actions/runs/13181421001/job/36792946491?pr=6365

Reply:
Tracked here: H-4028